src/IotHub/IotHub/help/Invoke-AzIotHubManualFailover.md

```diff
@@ -0,0 +1,183 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.IotHub.dll-Help.xml
+Module Name: Az.IotHub
+online version: https://docs.microsoft.com/en-us/powershell/module/az.iothub/invoke-aziothubmanualfailover
+schema: 2.0.0
+---
+
+# Invoke-AzIotHubManualFailover
+
+## SYNOPSIS
+Initiate a manual failover for the IoT Hub to the geo-paired disaster recovery region.
+
+## SYNTAX
+
+### ResourceSet (Default)
+```
+Invoke-AzIotHubManualFailover [-ResourceGroupName] <String> [-Name] <String> [-FailoverRegion <String>]
+ [-AsJob] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### InputObjectSet
+```
+Invoke-AzIotHubManualFailover [-InputObject] <PSIotHub> [-FailoverRegion <String>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdSet
+```
+Invoke-AzIotHubManualFailover [-ResourceId] <String> [-FailoverRegion <String>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+It will trigger the failover your IoT hub to the secondary location. This action will cause down time and telemetry loss to your solution. 
+This is a long running operation and could take several minutes to finish. Please exercise with caution when using it.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Invoke-AzIotHubManualFailover -ResourceGroupName "myresourcegroup" -Name "myiothub"
+```
+
+Initiating failover process of "myiothub" IoT Hub.
+
+### Example 2
+```powershell
+PS C:\> Invoke-AzIotHubManualFailover -ResourceGroupName "myresourcegroup" -Name "myiothub" -AsJob
+```
+
+Initiating failover process of "myiothub" IoT Hub in the background.
+
+### Example 3
+```powershell
+PS C:\> Invoke-AzIotHubManualFailover -ResourceGroupName "myresourcegroup" -Name "myiothub" -FailoverRegion "WestUS2"
+```
+
+Initiating failover process of "myiothub" IoT Hub to the specified geo-paired disaster recovery region.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -FailoverRegion
+The region that the IoT hubfails over to.
+Must be the paired region to the current IoT hub region.
```

Comment:
If `-FailoverRegion` must be the paired region, no other options, does it make sense to be a parameter?

Reply:
Correct, So removed it